cli/pkg/local/app.go

```diff
@@ -516,3 +525,29 @@ func initLogger(isVerbose bool, logFormat LogFormat) (logger *zap.Logger, cleanu
 		luLogger.Close()
 	}
 }
+
+func isExternalStorageEnabled(dbPath string, variables map[string]string) (bool, error) {
```

Comment:
Add a comment that this is for temporary backwards compatibility

Reply:
Theoretically this is not temporary, a project which was created before these changes will have to always operate with the older logic. Moving the comment below this line here.